src/main/java/de/medizininformatik_initiative/processes/common/fhir/client/FhirClientFactory.java

```diff
@@ -16,10 +16,10 @@
 import org.slf4j.LoggerFactory;
 
 import ca.uhn.fhir.context.FhirContext;
+import de.hsheilbronn.mi.utils.crypto.CertificateHelper;
+import de.hsheilbronn.mi.utils.crypto.io.CertificateReader;
+import de.hsheilbronn.mi.utils.crypto.io.PemIo;
```

Comment:
crypto-utils 4.0.0 can only be used if copied into the process plugin, see pom.xml comment

Reply:
reset with bf0f632